SourceX/miniwin/misc_msg.cpp

```diff
@@ -28,8 +34,13 @@ void SetCursorPos(int X, int Y)
 	mouseWarpingX = X;
 	mouseWarpingY = Y;
 	mouseWarping = true;
+#ifndef SWITCH

```

Comment:
This ifdef should not be necessary and is likely harmful

Reply:
No it isn't. The Window stuff doesn't work on Switch, AFAIK. Remember there is no SDL mouse pointer on Switch.

Reply:
Does `SDL_WarpMouseInWindow` not generate an event on the Switch?
If it does, this event is no longer necessary. Note that the rest of the code changed quite a bit since the original Switch port.
If it doesn't, let's add a comment explaining why we do this.
Anyway, worth checking.